tuning/config/configs.py

```diff
@@ -38,6 +38,19 @@ class ModelArguments:
         metadata={"help": "Use Flash attention v2 from transformers, default is True"},
     )
     torch_dtype: Optional[Union[torch.dtype, str]] = torch.bfloat16
+    custom_tokenizer_name_or_path: Optional[str] = field(
```

Comment:
Is it not possible to use `tokenizer_name_or_path`. Is it not possible to bypass the issue with PEFTConfig?

Reply:
yes i agree, best is to not have 2 flags with does basically the same thing. This will involve a design change, so best to discuss with maintainers.